Switch assessment

eedrummer opened the pull request:
Allows users to switch the risk assessment being displayed on the patient viewer

eedrummer committed:
[WIP] Starting work on allowing the switching of assessments

eedrummer committed:
Implemented the switching of risks on the patient viewer

noranda commented on app/components/patient-summary.js:
```diff
@@ -2,7 +2,26 @@ import Ember from 'ember';
 
-export default PatientSummaryComponent;
\ No newline at end of file
+export default PatientSummaryComponent;
```
For consistency, this export should be up at the top. Remove the `let` statement on line 3 and replace it with:

``` js
export default Ember.Component.extend({
```

noranda commented on app/components/patient-viewer.js:
```diff
@@ -2,7 +2,29 @@ import Ember from 'ember';
 
-export default PatientViewerComponent;
\ No newline at end of file
+export default PatientViewerComponent;
```
Same here, instead of assigning the component to a variable just export it directly:

``` js
export default Ember.Component.extend({
```

eedrummer committed:
Exporting directly as opposed to assigning, then exporting

We are now going to place all export statements at the top of our
JavaScript files.

eedrummer commented:
@noranda I think I've addressed your comments. Please let me know if I missed something. Thanks for helping me to keep my code in line with the style I just advocated for. :smile:
